pkg/controllers/localbuild/gitea.go

```diff
@@ -43,7 +45,7 @@ func RawGiteaInstallResources(templateData any, config v1alpha1.PackageCustomiza
 	return k8s.BuildCustomizedManifests(config.FilePath, "resources/gitea/k8s", installGiteaFS, scheme, templateData)
 }
 
-func giteaAdminSecretObject() corev1.Secret {
+func (r *LocalbuildReconciler) GiteaAdminSecretObject() corev1.Secret {
```

Comment:
This doesn't need to be a method. Better to move to the utils package because it's probably needed in the get command.

Reply:
What do you think about this f0754f7 ?